src/Security/Security.php

```diff
@@ -47,9 +48,19 @@ public function isRequestValid(): bool
 		$key = SettingsSecurity::SETTINGS_SECURITY_DATA_KEY;
 		$keyName = UtilsSettingsHelper::getOptionName($key);
 		$data = UtilsSettingsHelper::getOptionValueGroup($key);
-		$ip = $this->getIpAddress(true);
+		$ip = $this->getIpAddress();
 		$time = \time();
 
+		// Bailout if the IP is in the ignore list.
+		$ignoreIps = Components::flattenArray(UtilsSettingsHelper::getOptionValueGroup(SettingsSecurity::SETTINGS_SECURITY_IP_IGNORE_KEY));
+
+		if (\array_key_exists($ip, \array_flip($ignoreIps))) {
+			return true;
+		}
```

Comment:
Could this be written with `isset`?

```suggestion
		if (isset(\array_flip($ignoreIps)[$ip])) {
			return true;
		}
```

Not sure what the `$ignoreIps` array looks like, but unless the array is like:

```php
[
  '123.456.789.0' => null,
   ...
]
```

You can use `isset`. I kinda reckon that since you are flipping the array you'll never have null inside (otherwise you'd loose the key that is `null`).

Reply:
sure looks good